clang/lib/Sema/SemaTemplate.cpp

```diff
@@ -2886,21 +2930,6 @@ void DeclareImplicitDeductionGuidesForTypeAlias(
     // parameters, used for building `TemplateArgsForBuildingFPrime`.
     SmallVector<TemplateArgument, 16> TransformedDeducedAliasArgs(
         AliasTemplate->getTemplateParameters()->size());
```

Comment:
I see this was just moved... I'd still love to see us figure out WHEN this could be nullptr.  I fear this is a case where we are going to 'miss' something if we add a new template param type.